content/smart.md

```diff
@@ -89,7 +90,8 @@ Cerner understands that it's counter-intuitive to prompt the user for their cred
 
 ### Embedded Browser Control ###
 
-There have been a lot of questions and interests around what embedded browser control Cerner's implementation uses. In this section, you'll learn more about the embedded browser control. The actual browser control that we use is [IWebBrowser2 C++ interface](https://msdn.microsoft.com/en-us/library/aa752127(v=vs.85).aspx) by Microsoft. Please take a look at the documentation for more information. With this embedded browser control, there are some limitations compared to the stand-alone IE browser. See below for these limitations.
+There have been a lot of questions and interests around what embedded browser control Cerner's implementation uses. In this section, you'll learn more about the embedded browser control. The actual browser controls that we use are  [IWebBrowser2 C++ interface](https://msdn.microsoft.com/en-us/library/aa752127(v=vs.85).aspx) for IE and [WebView2 control](https://docs.microsoft.com/en-us/microsoft-edge/webview2/) for Edge browser both developed by Microsoft. Please take a look at the documentation for more information. With this embedded browser control, there are some limitations compared to the stand-alone IE or Edge browser. See below for these limitations.
```

Comment:
we also mention the IE control in the section above but not Edge's. we should be consistent on where info is included (or if it should be duplicated)

Reply:
It is mentioned there: 'and [WebView2 control](https://docs.microsoft.com/en-us/microsoft-edge/webview2/) for Edge browser both developed by Microsoft'